2018-03-25 20:57:44,582 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:14,924 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:45]
2018-03-25 20:59:16,145 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:16,145 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:16,216 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:17,904 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:17,959 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:17,959 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,468 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,468 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,519 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,576 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,576 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,636 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,636 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,696 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,696 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,797 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,886 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,887 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:19,966 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:19,966 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:20,033 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:20,034 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:20,100 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:20,100 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:41,392 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:45]
2018-03-25 20:59:42,481 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 20:59:42,482 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 20:59:42,550 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:00:55,291 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:00:55,291 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:00:55,348 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:05:31,305 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:05:31,365 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:05:31,365 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:05:47,128 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:05:47,184 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:05:47,184 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:05:47,284 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:05:47,284 INFO: load_user: user2 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:05,615 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:45]
2018-03-25 21:06:16,920 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:06:24,752 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:24,879 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:24,956 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:06:24,957 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:27,614 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:27,707 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:06:27,707 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:40,700 INFO: load_user: user1 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:06:40,780 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:07:10,111 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:45]
2018-03-25 21:07:20,047 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:07:24,939 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:07:25,061 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:07:25,135 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:07:25,135 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:07:28,262 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:07:28,332 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:07:28,332 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:17,356 INFO: Flask Starter startup [in /Users/patryan/Development/mygithub/flask-starter/app/__init__.py:45]
2018-03-25 21:08:20,563 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:20,713 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:08:20,714 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:24,738 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:24,840 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:08:24,841 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:32,567 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:32,650 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:08:32,650 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:38,363 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:38,449 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:08:45,476 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:45,599 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
2018-03-25 21:08:45,668 ERROR: File not found [in /Users/patryan/Development/mygithub/flask-starter/app/errors/handlers.py:7]
2018-03-25 21:08:45,669 INFO: load_user: user3 [in /Users/patryan/Development/mygithub/flask-starter/app/services/auth_services.py:12]
